loculus_values/values.yaml

```diff
@@ -2266,6 +2266,7 @@ organisms:
         configFile:
           <<: *preprocessingConfigFile
           nextclade_dataset_name: "nextstrain/rsv/a/EPI_ISL_412866"
+          nextclade_dataset_version: "2024-11-27--02-51-00Z"
```

Comment:
Where is this used in loculus? I can't see it in values.yaml or prepro config?

Reply:
You're right it isn't!

Reply:
so it should be `nextclade_dataset_tag` instead of `nextclade_dataset_version`?

Reply:
yep